src/services/token/queries/use-token.ts

```diff
@@ -57,11 +59,24 @@ export const useTokenList = (
 
 export const useTokenBalances = (
   params: IFetchTokenBalancesParams,
-  options?: UseQueryOptions<AssetBalance[] | null>
+  options: UseQueryOptions<AssetBalance[] | null> = {}
 ) => {
+  const {data: nativeToken, isFetched} = useBalance({
+    address: params.address as `0x${string}`,
```

Comment:
You can import the `Address` type from `wagmi`:
```suggestion
    address: params.address as Address,
```